commands/global/new.js

```diff
@@ -37,6 +38,11 @@ module.exports.handler = async function handler(options) {
       ${projectName} <TRANSFORM NAME> path/of/files/ or/some**/*glob.js
       \`\`\`
 
+      ## Local Usage
+      \`\`\`
+      npx ${process.cwd()}/${projectName}${cliPath} <TRANSFORM NAME> path/of/files/ or/some**/*glob.js
```

Comment:
Do we even need `npx` here? I _think_ this could be:

```suggestion
      node ./bin/cli.js <TRANSFORM NAME> path/of/files/ or/some**/*glob.js
```

Reply:
You're right we don't need npx here, I was using it for consistency with the other examples but using node is probably better for local invocation.